iOSClient/Settings/NCManageE2EE.swift

```diff
@@ -169,76 +169,81 @@ struct NCViewE2EE: View {
             if manageE2EE.isEndToEndEnabled {
 
                 List {
-
-                    Section(header: Text(""), footer: Text(manageE2EE.statusOfService + "\n\n" + "End-to-End Encryption " + NCGlobal.shared.capabilityE2EEApiVersion)) {
+                 
+                    Section(header: Text(""), footer: Text("End-to-End Encryption" + NCGlobal.shared.capabilityE2EEApiVersion))
+                    {
                         Label {
                             Text(NSLocalizedString("_e2e_settings_activated_", comment: ""))
                         } icon: {
-                            Image(systemName: "checkmark.circle.fill")
+                            Image("checkmark.circle.fill")
```

Comment:
Keeping the "systemName" cause any issue while loading the image? If no, then you can keep the systemName as it is.